.clang-tidy

```diff
@@ -12,11 +12,15 @@ Checks: '-*,
          -cppcoreguidelines-pro-type-reinterpret-cast,
          -cppcoreguidelines-pro-bounds-constant-array-index,
          llvm-*,
+         -llvm-header-guard,
+         -llvm-include-order,
          misc-*,
          -misc-no-recursion,
          -misc-non-private-member-variables-in-classes,
+         -misc-const-correctness,
          modernize-*,
          -modernize-avoid-c-arrays,
+         -modernize-use-nullptr,
```

Comment:
crashes `clang-tidy`

Reply:
maybe only in visual studio 2022, see [https://github.com/llvm/llvm-project/issues/63711](url)

Reply:
re-enabled, to be checked in Visual Studio Code